### DISCUSSION TITLE
Github Desktop issue

### DISCUSSION THREAD

**Discussion by sksksk2024** [community/community#137703] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hi, comunity! When I try to access the "open in VSCode" option from Github Desktop, it doesn't work. The error that pops up:
```
Error: spawn cmd.exe ENOENT
    at __node_internal_captureLargerStackTrace (node:internal/errors:563:5)
    at __node_internal_errnoException (node:internal/errors:690:12)
    at ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

Version: 3.4.3
OS: Windows 6.2.9200
```
 I've tried to verify the "code ." command from cmd(it works), check the paths from environment settings and looked up for info in the internet, but without any result!
Btw, I want to resolve the problem in order to get the "Pair Extraordinaire Badge"!
 Any advice is welcome, and thank you in advance! :)

**Comment by steveward** [comment, marked as answer]
Could you open the command prompt and share the output of echo `%PATH%`?

If `System32` is not in your `PATH`, you will see that `ENOENT` error. You will need to edit your `PATH` by following [this article](https://www.howtogeek.com/118594/how-to-edit-your-system-path-for-easy-command-line-access/) and adding the following line:

`C:\Windows\system32`

**Reply by sksksk2024** [reply]
You have no idea how grateful I am for your response("C:\Windows\system32" path resolved the issue)!!🤯
Thanks a ton, mate!!!🔥😊

**Reply by rksdutt** [reply]
This helps me as well to install Github Desktop.
